docs/a320-simvars.md

```diff
@@ -71,14 +71,6 @@
     - Position (0-2)
     - 0 is LO, 1 is NORM, 2 is HI
 
-- A32NX_AIRCOND_PACK1_FAULT
-    - Bool
-    - True if fault in pack 1
-
-- A32NX_AIRCOND_PACK2_FAULT
-    - Bool
-    - True if fault in pack 2
-
```

Comment:
**suggestion**: Consider adding the new names under `A32NX_OVHD_{name}_PB_HAS_FAULT` within this file.

Reply:
It was already there in line 2347 😃